phpstan-baseline.neon

```diff
@@ -795,6 +780,16 @@ parameters:
 			count: 1
 			path: tests/Sabre/Xml/ServiceTest.php
 
+		-
+			message: "#^Property Sabre\\\\Xml\\\\Service\\:\\:\\$elementMap \\(array\\<string, \\(callable\\(\\)\\: mixed\\)\\|class\\-string\\>\\) does not accept array\\{\\{http\\://sabre\\.io/ns\\}test\\: 'Test\\!'\\}\\.$#"
+			count: 1
+			path: tests/Sabre/Xml/ServiceTest.php
+
+		-
+			message: "#^Property Sabre\\\\Xml\\\\Service\\:\\:\\$namespaceMap \\(array\\<string, class\\-string\\>\\) does not accept array\\<string, string\\>\\.$#"
+			count: 4
+			path: tests/Sabre/Xml/ServiceTest.php
```

Comment:
these test-cases seem to intentionally use test-data which does not adhere to our own api?

Reply:
See #216 commit https://github.com/sabre-io/xml/pull/216/commits/517dd4866c1ddd3b77d7e48a84a2c04fb9ec52a3

These can be got rid of by specifying a string that PHPstan recognizes as a class name - I used the built-in `stdClass` 

That makes everything pass. Maybe that is OK to do in the tests. If you think so, then cherry-pick that commit into this PR.

That PR also fixes small things that cs-fixer complains about:
https://github.com/sabre-io/xml/pull/216/commits/631da960f71ae7c4117ac0d90b2d1efebde0e887

Reply:
picked the commit, thx